### PR TITLE
Task/add code errors

### LINKED ISSUE #91: Most of the errors don't have an associated code
Error objects can have an associated LWM2M status code, that will be given to the client as part of a response in an error case. But most of the errors don't have its code associated. This may be causing some error cases to return non-error (2.xx) codes. Tests should be created to uncover this situation and error objects completed to fix it.

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #91

### CODE REVIEW

**Comment by anabelengp** [comment]
LGTM (minor, missing newline at end of file)

File: lib/errors.js [hunk 1]
```diff
@@ -80,38 +83,47 @@ module.exports = {
...
         this.name = 'CLIENT_RESPONSE_ERROR';
         this.message = 'Error received while waiting for a client response: ' + msg;
+        this.code = '5.01';
     },
     ContentFormatNotFound: function() {
         this.name = 'CONTENT_FORMAT_NOT_FOUND';
         this.message = 'No Content Format list found.';
+        this.code = '4.00';
     }
 };
```
Comment (gtorodelvalle) [inline lib/errors.js]: As @anabelengp mentions in her comment, final blank line appreciated... :)

**Comment by gtorodelvalle** [comment]
I saw there are distinct errors with the same code... :) Just out of my curiosity, which was your criteria to set these codes? Shouldn't error codes be unique? :)

**Comment by dmoranj** [comment]
@gtorodelvalle Concerning the error codes, maybe it is not that clear what is their purpose. The `code` field indicates what COAP status code is to be returned if one of those errors is thrown when a request of a client is being attended; they are more or less the same as the HTTP status code (200, 400, 501). In fact, the semantics are mostly the same. I used the following criteria: 4.00 if there is something wrong with the user request, 5.01 if the problem is in the server. This error codes may be refined in future PRs, but the important part is to have an error code in every error, so there is something returned for each error.

File: lib/errors.js (hunk 1 above)
Comment (dmoranj) [inline lib/errors.js]: Fixed in c9b7dee

**Comment by gtorodelvalle** [comment]
LGTM ;)

**Pull request merged**